bw2io/extractors/ecospold2.py

```diff
@@ -32,6 +32,22 @@
 
 
 def getattr2(obj, attr):
+    """
+    Get attribute of an object; return empty dict if AttributeError occurs.
```

Comment:
Maybe link to documentation for `getattr()`?

Reply:
Inserted link to built-in getattr(). Not sure if this is what you mean. Please let me know if I misunderstood you.